docs/manual/external-tools.tex

```diff
@@ -725,10 +725,10 @@
 
 \begin{enumerate}
 
-\item Change the project SDK to 1.8, as explained at
+\item Change the project SDK to 11, as explained at
   \url{https://www.jetbrains.com/help/idea/sdk.html#change-project-sdk}.
 
-\item Change the language level for your project to 8, as explained at
+\item Make sure the language level for your project is 11 (after the last step, it should show up as your SDK default), as explained at
```

Comment:
Can you test if this works with the project SDK set to 11, but the language level set to 8?

Reply:
It does, but wouldn't it make sense to use the SDK default?

Reply:
Not for all projects.  Some projects use JDK 11 to build, but set the release level to a lower version, such as 8.  This allows the project to be run on Java 8+.  So the text ` Make sure the language level for your project is 11`  should be changed to ` Make sure the language level for your project is 8 or higher`.